docs/source/performance_benchmarks.md

```diff
@@ -0,0 +1,38 @@
+
+# Performance Benchmarks
+
+## Large Language Models**
+
+### Pretraining
+
+- The results in the table below show pre-training performance (using NeMo Framework 24.07) of various models on DGXH100, with FP8 precision.
+- Please refer to `MLCommons Training results <https://mlcommons.org/benchmarks/training/>`_ for performance of GPT3-175B pre-training on large scale H100 systems.
```

Comment:
Can we just add the performance MLPerf GPT3-1785B using the numbers and put the link directly on the number??